openapi/openapi/src/main/scala/endpoints4s/openapi/JsonSchemas.scala

```diff
@@ -130,6 +130,23 @@ trait JsonSchemas extends algebra.JsonSchemas with TuplesSchemas {
         }
     }
 
+    sealed abstract class RecursiveSchema extends DocumentedJsonSchema {
```

Comment:
Would it be possible to reuse `LazySchema` instead? (by adding a `def name: String` to it)

Reply:
Done